cetmix_tower_aws/models/cx_tower_command.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 Cetmix OÜ
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+from odoo.tools.safe_eval import wrap_module
+
+from . import constants
+
+# Wrap boto3 safely
+boto3 = wrap_module(__import__("boto3"), ["client", "resource", "Session"])
+
+
+class CxTowerCommand(models.Model):
+    """Extends cx.tower.command to add AWS boto3 functionality."""
+
+    _inherit = "cx.tower.command"
+
+    @api.model
+    def _get_eval_context(self, server=None):
+        """
+        Overrides _get_eval_context to add boto3 to the evaluation context.
+        """
+        eval_context = super()._get_eval_context(server=server)
+        eval_context["boto3"] = boto3
+        return eval_context
+
+    @api.depends("action")
```

Comment:
This function already [depends](https://github.com/cetmix/cetmix-tower/blob/0c8eb47fe53bbcbb2deef03cef43f6e254785dd1/cetmix_tower_server/models/cx_tower_command.py#L198) on action. No need to add this dependency once again.